doc/installation.rst

```diff
@@ -9,13 +9,15 @@ Composer_:
 
 .. code-block:: bash
 
-    $ composer require friendsofsymfony/http-cache:~1.0
+    $ composer require friendsofsymfony/http-cache
 
 .. note::
 
-    This library follows `Semantic Versioning`_. Because constraint ``~1.0``
-    will only increment the minor and patch numbers, it will not introduce BC
-    breaks.
+    This library follows `Semantic Versioning`_. Except for major versions, we
+    aim to not introduce BC breaks in new releases. You should still test your
+    application after upgrading though. What is a bug fix for somebody could
+    break something for others when they where (probably unawares) relying on
+    that bug.
```

Comment:
this is kind of stating the obvious. we can also remove everything after "not to introduce BC breaks in new releases" if you prefer